tutor4points/core/forms.py

```diff
@@ -142,6 +142,34 @@ class Meta:
     helper = FormHelper()
     helper.form_id = 'form'
 
+class RateTutorForm(forms.ModelForm):
+
+    class Meta:
+        model = Rating
+
+        # layout where want fields to be, type in order you want it to appear
+        fields = ('rating', 'comment','rating_type')
+
+        # customize placeholders
+        widgets = {
+            'rating':
+            forms.NumberInput(attrs={'placeholder': '1-5'}),
+            'comment':
+            forms.TextInput(attrs={'placeholder': 'comment here'}),
+            'Type of rating (tutor or tutee)':
+            forms.TextInput(attrs={'placeholder': 'comment'}),
+
+        }
+
+        # customize form labels
+        labels = {
+            'rating': "Enter a rating 1-5 stars",
+            'comment': "Explain how you're session went",
```

Comment:
I think you can just call this field "Comment" so the user don't feel that they have to type too much